aikau/src/main/resources/alfresco/renderers/_ActionsMixin.js

```diff
@@ -316,6 +316,64 @@ define(["dojo/_base/declare",
          {
             this.alfLog("log", "Skipping action as it's missing from whitelist: " + action);
          }
+      },
+
+      /**
+       * 
+       * @instance
+       * @since 1.0.62
+       */
+      createEmptyMenu: function alfresco_renderers__ActionsMixin__createEmptyMenu() {
+         this.dropDown = this.createWidget({
+            id: this.id + "_DROPDOWN",
+            name: "alfresco/menus/AlfMenuGroups",
+            config: {
+               widgets: [
+                  {
+                     name: "alfresco/menus/AlfMenuGroup",
+                     assignToScope: this,
+                     assignTo: "_menu"
+                  }
+               ]
+            }
+         });
+      },
+
+      /**
+       * 
+       * @instance
+       * @param  {Function} callback The function to call to display the drop-down
+       * @since 1.0.62
+       */
+      createDropDownMenu: function alfresco_renderers__ActionsMixin__createDropDownMenu(callback) {
+         this.createEmptyMenu();
+         this.addActions();
+         callback();
```

Comment:
Yes, because we're hooking into the way that Dojo provides the drop-down handling capabilities, see https://dojotoolkit.org/reference-guide/1.10/dijit/_HasDropDown.html for the details on the mixin we're leveraging and the recommendations for asynchronous loading of the menu.